darts/models/forecasting/regression_ensemble_model.py

```diff
@@ -123,6 +123,11 @@ def __init__(
                 lags=None, lags_future_covariates=[0], fit_intercept=False
             )
         elif isinstance(regression_model, RegressionModel):
+            raise_if_not(
+                regression_model.multi_models,
+                "`regression_model.multi_models = False` is not supported for `RegressionEnsembleModel.",
```

Comment:
Maybe below could be more concise
```suggestion
                "Cannot use `regression_model` that was created with `multi_models = False`.",
```